framework/src/test/java/tests/StubParserEnumTest.java

```diff
@@ -0,0 +1,24 @@
+package tests;
+
+import java.io.File;
+import java.util.List;
+import org.checkerframework.framework.test.FrameworkPerDirectoryTest;
+import org.junit.runners.Parameterized.Parameters;
+
+public class StubParserEnumTest extends FrameworkPerDirectoryTest {
```

Comment:
I would also move this test to a jtreg test. Many other stub tests are in `checker/jtreg/stubs/`.